src/components/TextField/TextField.tsx

```diff
@@ -122,6 +122,10 @@ const getUniqueID = createUniqueIDFactory('TextField');
 class TextField extends React.PureComponent<Props, State> {
   state: State = { height: null, value: '' };
 
+  static defaultProps: any = {
+    type: 'text'
```

Comment:
@emgage-smit do we want to apply this by default? I thought we decided we wanted to force the dev to select a type value, in which case we would not want a default.